examples/gallery/lines/envelope.py

```diff
@@ -0,0 +1,79 @@
+"""
+Envelope
+--------
+The ``close`` parameter of the :meth:`pygmt.Figure.plot` method can be
+used to build a symmetrical or an asymmetrical envelope. The user can
+give either the deviations or the bounds in y direction. For the first
+case append ``"+d"`` or ``"+D"`` and for the later case ``"+b"``.
+"""
+
+
+import pandas as pd
+import pygmt
+
+# Define pandas DataFrame with columns for x and y as well as the
+# lower and upper deviations
+df = pd.DataFrame(
+    data={
+        "x": [1, 3, 5, 7, 9],
+        "y": [0.5, -0.7, 0.8, -0.3, 0.1],
+        "y-error-low": [0.2, 0.2, 0.3, 0.4, 0.2],
+        "y-error-upp": [0.1, 0.3, 0.2, 0.4, 0.1],
+    }
+)
+
+# Create Figure instance
+fig = pygmt.Figure()
+
+fig.basemap(
+    region=[0, 10, -1.5, 1.5],
+    projection="X10c",
+    frame="a1f1g1",
```

Comment:
Would remove the grid:
```suggestion
    frame="a1",
```